v0/destinations/profitwell/utils.js

```diff
@@ -0,0 +1,199 @@
+const { send } = require("../../../adapters/network");
```

Comment:
`send` is not there in anymore. Need to update the code.

Reply:
updated with `httpGET`